spec/lib/card/content_spec.rb

```diff
@@ -89,7 +89,7 @@
     "\n        More urls: ",
     "<a class=\"external-link\" href=\"http://wagn.com/a/path/to.html\">wagn.com/a/path/to.html</a>",
     "\n        ",
-    "<a class=\"external-link\" href=\"http://localhost:2020/path?cgi=foo&bar=baz\">http://localhost:2020/path?cgi=foo&bar=baz</a>",
+    "<a class=\"external-link\" href=\"http://localhost:2020/path?cgi=foo+bar=baz\">http://localhost:2020/path?cgi=foo+bar=baz</a>",
```

Comment:
I'm not totally positive what is correct for this.  It is entity encoding the & now, and it wasn't before.  I think this is related to using build_link vs. that other odd method (link_to_page ?)  I think it get's interpreted the same way encoded or not.